doc/source/data/doc_code/transforming_datasets.py

```diff
@@ -77,29 +77,80 @@ def transform_batch(df: pandas.DataFrame) -> pandas.DataFrame:
 # fmt: on
 
 # fmt: off
-# __writing_native_udfs_begin__
+# __writing_default_udfs_tabular_begin__
 import ray
 import pandas as pd
 
 # Load dataset.
 ds = ray.data.read_csv("example://iris.csv")
+ds.default_batch_format()
+# pandas.core.frame.DataFrame
 
 # UDF as a function on Pandas DataFrame batches.
-def pandas_transform(df: pd.DataFrame) -> pd.DataFrame:
+def pandas_transform(df_batch: pd.DataFrame) -> pd.DataFrame:
     # Filter rows.
-    df = df[df["variety"] == "Versicolor"]
+    df_batch = df_batch[df_batch["variety"] == "Versicolor"]
     # Add derived column.
-    df["normalized.sepal.length"] =  df["sepal.length"] / df["sepal.length"].max()
+    # Notice here that `df["sepal.length"].max()` is only the max value of the column
+    # within a given batch (instead of globally)!!
+    df_batch["normalized.sepal.length"] = df_batch["sepal.length"] / df_batch["sepal.length"].max()
```

Comment:
Nit: Using `.loc[:, col_name]` is the idiomatic Pandas column setter, since it should be more efficient and avoids accidental chained indexing, so we try to do this in all UDF examples:
```suggestion
    df_batch.loc[:, "normalized.sepal.length"] = df_batch["sepal.length"] / df_batch["sepal.length"].max()
```

Reply:
Ah good to know!!
I also updated another example down below.